legislation-parameters.md

```diff
@@ -0,0 +1,109 @@
+
+Coding a parameter means defining a list of values, and for each value the date where it starts to hold.
+
+Example: the threshold of the [`decote`](https://legislation.openfisca.fr/parameters/impot_revenu.decote.seuil)
+
+```xml
+  <NODE code="decote" description="Décote" origin="openfisca">
+    <CODE code="seuil" description="Seuil de la décôte" format="integer" origin="openfisca" type="monetary">
+      <VALUE deb="2014-01-01" valeur="1016" />
+      <VALUE deb="2013-01-01" valeur="1016" />
+      <VALUE deb="2012-01-01" valeur="960" />
+      <VALUE deb="2010-01-01" valeur="878" />
+```
+
+### The tag `<END>`
+
+When a parameter is removed from the legislation, a tag `<END>` is used.
+
+Example: if `decote` is removed from legislation in 2017, the previous example would become
+
+```xml
+  <NODE code="decote" description="Décote" origin="openfisca">
+    <CODE code="seuil" description="Seuil de la décôte" format="integer" origin="openfisca" type="monetary">
+      <END deb="2017-01-01" />
+      <VALUE deb="2014-01-01" valeur="1016" />
+      <VALUE deb="2013-01-01" valeur="1016" />
+      <VALUE deb="2012-01-01" valeur="960" />
+      <VALUE deb="2010-01-01" valeur="878" />
+```
+
+### The tag `<PLACEHOLDER>`
+
+When a value is expected to change at a given date but the value is not published yet, a tag `<PLACEHOLDER>` is used.
+
+Example: if the threshold for `decote` is expected to change in 2018, the previous example would become
+
+```xml
+  <NODE code="decote" description="Décote" origin="openfisca">
+    <CODE code="seuil" description="Seuil de la décôte" format="integer" origin="openfisca" type="monetary">
+      <END deb="2017-01-01" />
+      <PLACEHOLDER deb="2018-01-01" />
+      <VALUE deb="2014-01-01" valeur="1016" />
+      <VALUE deb="2013-01-01" valeur="1016" />
+      <VALUE deb="2012-01-01" valeur="960" />
+      <VALUE deb="2010-01-01" valeur="878" />
+```
+
+The script [`find_placeholders.py`](https://github.com/openfisca/openfisca-core/tree/master/openfisca_core/scripts/find_placeholders.py) finds all placeholders in an legislation. It may be used periodically to alert on parameters about the change:
+
+```
+$ python openfisca_core/scripts/find_placeholders.py /data/projects/openfisca-dummy-country
+find_placeholders.py : Find nodes PLACEHOLDER in xml parameter files
+Usage :
+    python find_placeholders /dir/to/search
+
+File /data/projects/openfisca-dummy-country/openfisca_dummy_country/parameters/param_more.xml
+
+
+File /data/projects/openfisca-dummy-country/openfisca_dummy_country/parameters/param_root.xml
+2012-01-01 root.impot.bouclier
+2016-01-01 root.contribution_sociale.salaire.bareme.tranche1
+2016-01-01 root.contribution_sociale.salaire.bareme.tranche1
+```
+
+## Query
+
+> Warning : this part of the documentation may be outdated as a new syntax was recently introduced : `legislation(period).path.to.parameter`
+
+Formulas retrieve parameters using the `simulation.legislation_at(instant)` method.
```

Comment:
I'm not sure this part should be in the user manual. It feels more like code comments that should be extracted into the reference doc.